R/location_methods.R

```diff
@@ -131,7 +135,8 @@ add_summary_location_row <- function(loc,
 add_grand_summary_location_row <- function(loc,
                                            data,
                                            style,
-                                           df_type = "styles_df") {
+                                           df_type = "styles_df",
+                                           in_stub = FALSE) {
```

Comment:
Let's infer `in_stub` from the class of `loc`

Reply:
Now done.